uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/leave-request-api_test.js

```diff
@@ -115,5 +147,259 @@ define([
         $httpBackend.flush();
       });
     });
+
+    describe('calculateBalanceChange()', function () {
+      var promise, requestData;
+
+      beforeEach(function () {
+        requestData = mockData.calculateBalanceChangeRequest();
+        spyOn(LeaveRequestAPI, 'calculateBalanceChange').and.callThrough();
+        promise = LeaveRequestAPI.calculateBalanceChange(requestData);
+      });
+
+      afterEach(function () {
+        $httpBackend.flush();
+      });
+
+      it('calls endpoint', function () {
+        promise.then(function (result) {
+          expect(LeaveRequestAPI.calculateBalanceChange).toHaveBeenCalled();
```

Comment:
What does this test? You have just called manually `calculateBalanceChange`, what's the point if the method has actually been called? You did that manually!